aws/resource_aws_glue_catalog_database_test.go

```diff
@@ -208,6 +257,21 @@ resource "aws_glue_catalog_database" "test" {
 `, rInt, desc)
 }
 
+func testAccGlueCatalogDatabase_validation(rName string) string {
```

Comment:
For clarity, it might be better if this was named against the attribute its testing:

```suggestion
func testAccGlueCatalogDatabaseConfigName(rName string) string {
```